### DISCUSSION TITLE
After I upgraded my self-hosted  Airbyte to the latest version (0.50.32) only full sync works

### DISCUSSION THREAD

**Discussion by maxacarvalho** [airbytehq/airbyte#31785] [General]
I had the sync between MySQL and BigQuery working for quite a long time.
Nothing fancy, just a simply sync with not a huge amount of data. I was using the incremental sync with deduplication.

Today I had to update the schema to include a few extra columns and everything stopped working.
The sync wouldn't work anymore. I started to see an error `pickling client objects is explicitly not supported.`.

So I went ahead and upgraded my Airbyte version. I also upgraded the connectors and sources.
Then I clean up my BigQuery data, and started over.

I'm able to sync data using the "full refresh | overwrite|" but none of the incremental methods work.

The error I'm getting tells me only "java.lang.UnsupportedOperationException: Raw state retrieval not supported by global state manager."

One thing to note. Using "Normalized tabular data" also triggers an exception so I have to use "Raw data (JSON)" instead.

I;m out of options, any suggestions?

**Comment by marcosmarxm** [comment]
@maxacarvalho can you confirm you continue to use the same Bigquery version from your previous platform version?

**Comment by maxacarvalho** [comment]
Hi @marcosmarxm 

You mean the destination connector? If so no, I upgraded everything.

**Comment by cbuckle1** [comment]
Having this same issue with Airbyte 0.50.30 for MySQL v3.1.6 --> Snowflake v3.4.9; it looks like if we run Full Refresh it runs CREATE STAGE IF NOT EXISTS... and then drops it at the end of the sync, but with incremental it jumps straight to UPDATE "airbyte_internal"."[schema]_raw__stream_[table]" SET "_airbyte_loaded_at" = NULL; since the stage does not exist it fails.

[a comment by an automated account is not shown]
